torch/csrc/jit/ir/irparser.cpp

```diff
@@ -306,6 +338,29 @@ void IRParser::parseAttr(Node* n) {
       default:
         throw ErrorReport(L.cur().range) << "Unexpected attr type";
     }
+  } else if (L.cur().text() == "annotate") {
+    L.next();
+    L.expect('(');
+    auto type = L.cur().text();
+    if (type != "List" && type != "Dict") {
+      throw ErrorReport(L.cur().range)
+          << "Unexpected annotation (only List and Dict can be parsed)";
+    }
+    L.next();
+    // use the profiling annotation instead of the `annotate()` type.
```

Comment:
profiling is too specific, since types could come elsewhere. I would say something like `ignore the annotations on the constants IValue, and recover type from the Node output`